app/javascript/packs/application.js

```diff
@@ -0,0 +1,17 @@
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+require("@rails/ujs").start()
+require("turbolinks").start()
+require("@rails/activestorage").start()
+require("channels")
```

Comment:
Remove all the comment lines
![image](https://user-images.githubusercontent.com/65896888/91243444-3f278a80-e774-11ea-9970-5ec779227af6.png)